docs/userGuide/usingComponents.md

```diff
@@ -205,4 +205,47 @@ If you want to include a keyword without rendering it to the page, add `d-none`
 This is good for catching <span class="keyword">regressions</span>
 ```
 
+# Using Tags
+
+You can use tags to selectively filter certain elements on web pages.

```

Comment:
"You can use tags to selectively filter ~`certain`~`HTML` elements ~`on web pages`~`when building a site`."